packages/backend/src/config/app.ts

```diff
@@ -39,9 +44,9 @@ export const Config = {
     process.env.NODE_ENV === 'test'
       ? `${process.env.DATABASE_URL}_${process.env.JEST_WORKER_ID}`
       : envString(
-          'DATABASE_URL',
-          'postgresql://postgres:password@localhost:5432/development'
-        ),
+        'DATABASE_URL',
+        'postgresql://postgres:password@localhost:5432/development'
+      ),
```

Comment:
Can you double check your prettier settings, or just run `pnpm format`? For me, it reverts back to the original formatting